src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -828,7 +828,7 @@ private async Task ResyncPeersAsync(List<int> peerIds)
                     {
                         this.logger.LogTrace("Resyncing peer ID {0}.", peerId);
 
-                        Task task = peer.Behavior<ConsensusManagerBehavior>().ResetPeerTipInformationAndSyncAsync();
+                        Task task = peer.Behavior<ConsensusManagerBehavior>().ResetPeerTipInformationAndSync();
```

Comment:
This seems wrong, if we want to do this in parallel we need to put the call to `ResetPeerTipInformationAndSync` in a new Task.

Reply:
I will revert this.